swarm/network/stream/snapshot_retrieval_test.go

```diff
@@ -154,15 +154,15 @@ func runFileRetrievalTest(nodeCount int) error {
 	//array where the generated chunk hashes will be stored
 	conf.hashes = make([]storage.Address, 0)
 
+	ctx, cancelSimRun := context.WithTimeout(context.Background(), 3*time.Minute)
```

Comment:
3 minutes? Why do we need that much time?

Reply:
i thought, if there are 128 nodes and 100 messages, we might need more time.

Reply:
you are going to change that anyway

Reply:
But it doesn't mean that it will run for 3 minutes in any case, right?

I seem to remember @nonsense being pretty vocal about tests taking too long in general in swarm, and even did quite a bit of work a while back to reduce the total amount of time tests takes. Anyway, my view is if it's a 3 minute long test, then that test should be classified as `--longrunning`

Reply:
let us refactor it after your PR is merged

Reply:
I'm not sure I understand; what will change in this particular context after my PR is merged...?

Reply:
right. your PR overrides the timeout. no need to refactor.

Reply:
Now I understand even less :)